libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMap.java

```diff
@@ -324,8 +325,8 @@ public void resumeCamera(@NonNull Location location) {
    * @param trackingCameraMode the tracking mode
    */
   public void resetCameraPositionWith(@NavigationCamera.TrackingMode int trackingCameraMode) {
+    updateLocationIconPosition(false);
```

Comment:
should probably use `mapPaddingAdjustor.isCentered()` instead of `false`? Otherwise, clicking the `recenterButton` will reset the padding.

Reply:
@hurrba yeah great catch, thanks!